modules.d/40network/net-lib.sh

```diff
@@ -263,10 +263,12 @@ ibft_to_cmdline() {
                     case "$vlan" in
                         [0-9]*)
                             echo "vlan=$dev.$vlan:$dev"
+                            dev="$dev.$vlan"
                             echo $mac > /tmp/net.${dev}.${vlan}.has_ibft_config
```

Comment:
Wouldn't that change:
 echo $mac > /tmp/net.${dev}.${vlan}.has_ibft_config
to
 echo $mac > /tmp/net.${dev}.${vlan}.${vlan}.has_ibft_config

effectively?

Reply:
Is a command line  like "ip=ibft0.666:dhcp" allowed at all?
That would mean reconfiguring the interface that was initially set up via firmware.
If the original ip setup from firmware gets overwritten by dhcp, is the interface still to be called ibft?